lib/miou.mli

```diff
@@ -1175,12 +1172,13 @@ module Mutex : sig
       will restart. The mutex must have been previously locked by the thread
       that calls {!val:unlock}.
 
-      @raise Sysy_error was not raised when unlocking an unlocked mutex or when
-      unlocking a mutex from a different task. *)
+      @raise Sysy_error
```

Comment:
typo here but can be fixed later.

Reply:
Indeed - this one was caught as part of #54.